internal/assets/contents/com.activestate.platform.state.plist.tpl

```diff
@@ -3,23 +3,17 @@
 <plist version="1.0">
     <dict>
         <key>Label</key>
-        <string>my.everydaytasks</string>
+        <string>com.activestate.StateToolService</string>
         <key>ProgramArguments</key>
         <array>
-            <!-- Wrap in `sh -c` so that $HOME is expanded -->
-            <string>sh</string>
-            <string>-c</string>
-            {{- if .Args }}
-            <string>{{.Exec}} {{.Args}}</string>
-            {{- else}}
-            <string>{{.Exec}}</string>
-            {{- end}}
+          <string>{{.Exec}}</string>
+          {{- if .Args }}
+          <string>{{.Args}}</string>
+          {{- end}}
         </array>
-        <key>ProcessType</key>
-        <string>Interactive</string>
```

Comment:
We also use this file to create the state-tray autostart. I think that one likely relies on this setting. Should make this a conditional instead perhaps?

The other changes are probably fine for state-tray, but to be safe we should verify that.

Reply:
Okay, currently this file is only being used by state-svc autostart. Whenever we go back to adding support for state-tray, then it can start utilizing this file too, as well as a conditional property I added.

Reply:
> Whenever we go back to adding support for state-tray

The state-tray is "supported", it's just in beta. Frankly, we probably should remove it but that's not for this ticket to address. Point is it's part of our codebase right now and thus we should address it.

Reply:
I built the state-tray macOS app bundle and it has no mention of ProcessType in its plist file, so it doesn't look like it references the plist template file in this PR. I have removed the conditional I added in the previous commit.